.github/workflows/llvm.yml

```
@@ -0,0 +1,55 @@
+name: LLVM Test
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SPEC_SPLIT_DOTS: 160
+
+jobs:
+  llvm_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        llvm_version: ["13.0.0", "14.0.0", "15.0.6"]
+    name: "Test LLVM ${{ matrix.llvm_version }}"
```

Stylistic: that would match job names in other worfkows.

```suggestion
    name: "LLVM ${{ matrix.llvm_version }}"
```